src/utils/translation.ts

```diff
@@ -0,0 +1,103 @@
+import _ from 'lodash';
+import VernaJSONSchemaType from '../types/rjsf';
+import { IntlFormatters } from '@formatjs/intl/src/types';
+import { Maybe } from '../types/utils';
+
+export const DEFAULT_PROPERTY_TRANSLATION = '';
+
+export function translateWidget(
+  widget: VernaJSONSchemaType,
+  formatMessage: IntlFormatters['formatMessage'],
+) {
+  function translateItem(item: VernaJSONSchemaType) {
+    return {
+      description: item.description
+        ? formatMessage({ defaultMessage: DEFAULT_PROPERTY_TRANSLATION, id: item.description })
+        : undefined,
+      enum: item.enum
+        ? item.enum.map((id: string) => {
+            return formatMessage({ defaultMessage: DEFAULT_PROPERTY_TRANSLATION, id: id });
+          })
+        : undefined,
+      title: item.title
+        ? formatMessage({ defaultMessage: DEFAULT_PROPERTY_TRANSLATION, id: item.title })
+        : undefined,
+    };
+  }
+
+  function translateItems(items: VernaJSONSchemaType | VernaJSONSchemaType[]) {
+    if (Array.isArray(items)) return items.map(translateItem);
+    else return translateItem(items);
```

Comment:
`else` is useless here
```suggestion
    return translateItem(items);
```